### DISCUSSION TITLE
Loosing Track of Series

### DISCUSSION THREAD

**Discussion by gdesanto** [advplyr/audiobookshelf#1873] [Ideas]
There are so many series I now watch for new releases on, that I loose track.

I wish there was some way to have an ongoing release list, or calendar to put new release dates on or something built in.

**Comment by advplyr** [comment]
There is a feature request open for this https://github.com/advplyr/audiobookshelf/issues/443
but unlikely this gets added until there is a reliable way to get this information.

**Comment by gdesanto** [comment]
Thanks for pointing out 443, I mean, yea automatically populating it would be amazing, but just a manual calendar of some kind so the user can populate release dates themselves would be handy for now.
